src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -307,6 +326,9 @@ const PIAIntakeFormPage = () => {
         if (e?.cause?.message === '409') {
```

Comment:
why not like the previous PR uses cause.status to check the error code

Reply:
bcz of code Conflict. the change is not part of this ticket. It will be `cause.status` eventually.